modules/effects/src/main/scala/dev/profunktor/redis4cats/algebra/strings.scala

```diff
@@ -26,6 +26,7 @@ trait StringCommands[F[_], K, V]
     with Increment[F, K, V]
     with Bits[F, K, V] {
   def del(key: K*): F[Unit]
+  def exists(key: K*): F[Long]
```

Comment:
I think it would be better if we return `F[Boolean]` since Redis would return either 1 if the key exists or 0 if it doesn't, and that can be easily matched to `true` and `false`.

On the other hand, I don't really like what Redis does with accepting multiple keys as input. It'll just return the sum of the partial results. For example:

```
redis> SET key1 "Hello"
"OK"
redis> EXISTS key1
(integer) 1
redis> EXISTS nosuchkey
(integer) 0
redis> SET key2 "World"
"OK"
redis> EXISTS key1 key2 nosuchkey
(integer) 2
```

I think we could do better and return `true` if ALL the keys exist and `false` if at least one key doesn't exist. Similar to what the `Monoid[All]` instance would do.

What do you think?

Reply:
makes sense